.github/workflows/publish-oss-for-cloud.yml

```diff
@@ -134,6 +140,13 @@ jobs:
           echo "GIT_REVISION=${GIT_REVISION}" >> $GITHUB_ENV
         shell: bash
 
+      - name: Prepare Runner for Building
+        uses: ./.github/actions/runner-prepare-for-build
+
+      - name: Prepare Docker context
+        run: VERSION=${{ needs.generate-tags.outputs.dev_tag }} SUB_BUILD=PLATFORM ./gradlew copyGeneratedTar airbyte-db:db-lib:copyInitSql
```

Comment:
maybe good idea to add a comment here saying this is required for buildx? Putting on my reader cap, I'm not sure this is 100% obvious without you and I discussing this yesterday.

Reply:
ah ok sure I'll add a comment

Reply:
added comments